solutions/standard/DA-types.md

```diff
@@ -0,0 +1,215 @@
+```hcl
+  {
+      "postgres_admin" : "Administrator",
+      "postgres_reader" : "Operator",
+      "postgres_viewer" : "Viewer",
+      "postgres_editor" : "Editor"
+  }
+```
+
+
+## Users <a name="users"></a>
+
+If you can't use the IAM-enabled `service_credential_names` input variable for access, you can create users and roles directly in the database. For more information, see [Managing users and roles](https://cloud.ibm.com/docs/databases-for-postgresql?topic=databases-for-postgresql-user-management&interface=ui).
+
+:exclamation: **Important:** The `users` input contains sensitive information (the user's password).
+
+- Variable name: `users`.
+- Type: A list of objects that represent a user
+- Default value: An empty list (`[]`)
+
+### Options for users
+
+- `name` (required): The username for the user account.
+- `password` (required): The password for the user account in the range of 10-32 characters.
+- `type` (required): The user type. The "type" field is required to generate the connection string for the outputs.
+- `role`: The user role. The role determines the user's access level and permissions.
+
+### Example users
+
+```hcl
+[
+  {
+    "name": "pg_admin",
+    "password": "securepassword123",
+    "type": "database",
+  },
+  {
+    "name": "pg_reader",
+    "password": "readpassword123",
+    "type": "ops_manager"
+  }
+]
+```
+
+## Autoscaling <a name="autoscaling"></a>
+
+The Autoscaling variable sets the rules for how database increase resources in response to usage. Make sure you understand the effects of autoscaling, especially for production environments. For more information, see [Autoscaling](https://cloud.ibm.com/docs/databases-for-postgresql?topic=databases-for-postgresql-autoscaling&interface=ui#autoscaling-consider).
+
+- Variable name: `auto_scaling`
+- Type: An object with `disk` and `memory` configurations
+
+### Disk options for auto_scaling
+
+Disk autoscaling specifies thresholds when scaling can occur based on disk usage, disk I/O utilization, or both.
+
+The disk object in the `auto_scaling` input contains the following options. All options are optional.
+
+- `capacity_enabled`: Whether disk capacity autoscaling is enabled (default: `false`).
+
+- `free_space_less_than_percent`: The percentage of free disk space that triggers autoscaling (default: `10`).
+
+- `io_above_percent`: The percentage of I/O (input/output) disk usage that triggers autoscaling (default: `90`).
+
+- `io_enabled`: Indicates whether IO-based autoscaling is enabled (default: `false`).
+
+- `io_over_period`: How long I/O usage is evaluated for autoscaling (default: `"15m"` (15 minutes)).
+
+- `rate_increase_percent`: The percentage increase in disk capacity when autoscaling is triggered (default: `10`).
+
+- `rate_limit_mb_per_member`: The limit in megabytes for the rate of disk increase per member (default: `3670016`).
+
+- `rate_period_seconds`: How long (in seconds) the rate limit is applied for disk (default: `900` (15 minutes)).
+
```

Comment:
Can the spacing in lists in this file be made consistent please. Most lists do not have blank lines, some do and some are mixed.

Reply:
sure thank you for the feedback, I have addressed this.